Documentation/api-reference.md

```diff
@@ -9,30 +9,33 @@ This document lists the APIs provided by AEP Roku SDK, along with code samples f
 - [sendEvent](#sendEvent)
 - [resetIdentities](#resetIdentities)
 - [(optional) setExperienceCloudId](#setExperienceCloudId)
+- [createMediaSession](#createMediaSession)
+- [sendMediaEvent](#sendMediaEvent)
 - [shutdown](#shutdown)
 
 ---
 
 ### AdobeAEPSDKInit
 
-The AEP task node performs the core logic of the SDK. If the task node instance is not provided, calling this function will create a new AEP task node instance and yield an SDK API instance.
-Then, the "getTaskNode" fucntion can be called via the API instance to retrieve the task node instance.
+> [!IMPORTANT]
+> The AEP task node performs the core logic of the SDK. Typically, a Roku project maintains only one instance of the AEP task node.
+
+It's recommended to first call AdobeAEPSDKInit() without passing an argument within the scene script. It initializes a new AEP task node and creates an associated SDK instance. Then, the task node instance can be retrieved via the getTaskNode() API.
```

Comment:
It is required right? Since if not then we need to provide the details on how to create their own taskNode of the type SDK requires. 

Can we reword

> [!WARNING]
> It's **required** to initialize the first SDK instance without taskNode. Use the taskNode instance created along with the first SDK instance then onwards.

Reply:
We do have a similar sample.

Reply:
And it's too much to add multiple alerts here. I'd prefer to have only one `important` alert.